assets/texts/pt-br/Changelog/next.md

```diff
@@ -0,0 +1 @@
+* **Corrigido:** Foi corrigido um bug que fazia com que o redirecionamento não fosse capturado no caso de versões mais recentes do Friendica. ([#62](https://github.com/rugk/mastodon-simplified-federation/issues/62))
```

Comment:
Note: Also feel free to already add the "translation added" line here (also in the English version of this file).

See https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#credit

Reply:
Will do.

Reply:
done by me now